packages/imperative/src/cmd/src/response/CommandResponse.ts

```diff
@@ -38,7 +38,7 @@ import { Logger } from "../../../logger";
 import { LoggerUtils } from "../../../logger/src/LoggerUtils";
 
 const DataObjectParser = require("dataobject-parser");
-
+export const DEFAULT_SPINNER_CHARS = "-oO0)|(0Oo-";
```

Comment:
Would we deprecate this property if we move the custom spinner implementation into public methods of the `.response.progress` getter?